src/ocrd/mets_server.py

```diff
@@ -556,17 +580,22 @@ async def add_file(
             # Add to workspace
             kwargs = file_resource.dict()
             workspace.add_file(**kwargs, force=force)
-            return file_resource
+            response = file_resource
+            self.log.info(f"POST /file -> {response.__dict__}")
+            return response
 
         # ------------- #
 
         if self.is_uds:
             # Create socket and change to world-readable and -writable to avoid permission errors
             self.log.debug(f"chmod 0o677 {self.url}")
             server = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
+            # TODO: Not required after #1284, consider removing
+            """
             if Path(self.url).exists() and not is_socket_in_use(self.url):
                 # remove leftover unused socket which blocks startup
                 Path(self.url).unlink()
+            """
```

Comment:
```suggestion
```

Then let's remove it, it was a workaround anyway.

Reply:
Yes, but @joschrew should potentially first confirm that with his docker setup. I did not want to remove that since he added the workaround and I could not test with his setup.